app/models/item_order.rb

```diff
@@ -15,4 +17,10 @@ def change_order_to_packaged
       order.packaged!
     end
   end
+
+  def change_to_unfulfilled
+    if self.fulfilled?
+      item.update(inventory: (item.inventory + quantity))
+    end
```

Comment:
I'm a little confused about how the order of these things works, but the test looks solid so I guess it's working! (and I was there when we designed this, ha!)